src/main/kotlin/model/NationalPrisonRadio.kt

```diff
@@ -15,7 +14,6 @@ class NationalPrisonRadio private constructor() {
         "The national radio station for prisoners. Made by prisoners, for prisoners"
       ).apply {
         Tags.PROVIDER.addTo(this)
-        setLocation(Location.External)
```

Comment:
Does this need `OutsideHMPPS.addTo(this)` adding here?

Reply:
Using `Tags.PROVIDER` adds it by default

https://github.com/ministryofjustice/hmpps-architecture-as-code/blob/bf29c7d33390004c631a8a50bbb5a868dd867059/src/main/kotlin/Tags.kt#L5-L13

I wonder if this is surprising/bad?

Reply:
Does this remove our need for the `PROVIDER` tag? We can pick this up elsewhere!